packages/core/src/Stores/client-store.js

```diff
@@ -2238,13 +2238,18 @@ export default class ClientStore extends BaseStore {
                 const is_account_param = account_keys.some(
                     account_key => key?.includes(account_key) && key !== 'affiliate_token'
                 );
+                const auth_keys = ['acct', 'token'];
+                const is_acct_token_params = auth_keys.some(
+                    account_key => key?.includes(account_key) && key !== 'affiliate_token'
+                );
```

Comment:
Just check if query parameters has `acct` or `token`, then we set `logged_state` to `true`